projects/burnr/src/hooks/api/useApiCreate.ts

```diff
@@ -11,7 +11,7 @@ import { LazyProvider } from '../../utils/types';
 import { useIsMountedRef, useLocalStorage } from '..';
 import westend from '../../assets/westend.json';
 
-console.log('ALL_PROVIDERS', ALL_PROVIDERS)
+console.log('ALL_PROVIDERS: ', ALL_PROVIDERS)
```

Comment:
leftover `console.log` ?

Reply:
This is there since beginning of project. For now I would leave it as its info that can be valuable.

Reply:
yeah we need a shared debugging library for genuinely useful debug messages that only appear when in local dev.  Ive added a `debug()` function to the extension for this